deployment/terraform/modelservice/main.tf

```diff
@@ -12,6 +12,10 @@ data "template_file" "config_data" {
   vars {
     db_endpoint = "${var.db_endpoint}"
     graphite_url = "${var.graphite_url}"
+    detector_mapper_es_urls = "${var.detector_mapper_es_urls}"
+    detector_mapper_es_config_vars_json = "${var.detector_mapper_es_config_vars_json}"
+    detector_mapper_index_name = "{var.modelservice["detector_mapper_index_name"]}"
+    detector_mapper_doctype = "{var.modelservice["detector_mapper_doctype"]}"
```

Comment:
Lines 17 and 18 are missing leading `$`.

Reply:
Fixed.